src/main/java/az/keytd/expensetracker/service/MailSenderService.java

```diff
@@ -0,0 +1,20 @@
+package az.keytd.expensetracker.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailSenderService {
```

Comment:
Also create entity end save every sent email to DB
![image](https://github.com/qismetkamilovv/expense-tarcker/assets/19332183/586d4240-e69f-4e43-bbcc-b4d3ff4076ce)

Reply:
create above table and its entity